### PR TITLE
feat(golang): add table scaleway_iam_api_key

### PR DESCRIPTION
**Pull request by jplanckeel**
# Describe

We add a table to list API Key and list users.  

API to list iam_apikey need organization_id, for this we add in config file organization_id. We have no other way to get this id.

# Example query results
<details>
  <summary>Results</summary>
  
```
select
  access_key,
  created_at,
  user_id,
  expires_at,
  default_project_id
from
  scaleway_iam_api_key
```

iam user 

```
select
  email,
  created_at,
  last_login_at,
  id,
  status,
  two_factor_enabled
from
  scaleway_iam_user
```
</details>

### CODE REVIEW

File: config/scaleway.spc [hunk 1]
```diff
@@ -6,6 +6,9 @@ connection "scaleway" {
   # variables using the `SCW_ACCESS_KEY` and `SCW_SECRET_KEY` arguments.
   access_key = "YOUR_ACCESS_KEY"
   secret_key = "YOUR_SECRET_ACCESS_KEY"
+  
+  # Your organization ID is the identifier of your account inside Scaleway infrastructure.
+  organization_id = "YOUR_ORGANIZATION_ID"
```
Comment (francois2metz) [inline config/scaleway.spc:11]: Could you describe how to get the organization id?

File: go.sum [hunk 2]
```diff
@@ -381,6 +381,8 @@ github.com/rogpeppe/go-internal v1.3.0/go.mod h1:M8bDsm7K2OlrFYOpmOWEs/qY81heoFR
 github.com/rogpeppe/go-internal v1.6.1 h1:/FiVV8dS/e+YqF2JvO3yXRFbBLTIuSDkuC7aBOAvL+k=
 github.com/scaleway/scaleway-sdk-go v1.0.0-beta.12 h1:Aaz4T7dZp7cB2cv7D/tGtRdSMh48sRaDYr7Jh0HV4qQ=
 github.com/scaleway/scaleway-sdk-go v1.0.0-beta.12/go.mod h1:fCa7OJZ/9DRTnOKmxvT6pn+LPWUptQAmHF/SBJUGEcg=
+github.com/scaleway/scaleway-sdk-go v1.0.0-beta.14 h1:yFl3jyaSVLNYXlnNYM5z2pagEk1dYQhfr1p20T1NyKY=
+github.com/scaleway/scaleway-sdk-go v1.0.0-beta.14/go.mod h1:fCa7OJZ/9DRTnOKmxvT6pn+LPWUptQAmHF/SBJUGEcg=
```
Comment (francois2metz) [inline go.sum:385]: Could you run `go mod tidy` to cleanup previous releases from the `go.sum`

**Comment by cbruno10** [comment]
@jplanckeel Sorry for the delay in review, I should get a chance later this week!

**Comment by jplanckeel** [comment]
Hi, no problem. I check this MR today to test/update with new version of steampipe ;)

File: scaleway/service.go [hunk 3]
```diff
@@ -44,6 +44,11 @@ func getSessionConfig(ctx context.Context, d *plugin.QueryData) (*scw.Client, er
 		secretKey = *scalewayConfig.SecretKey
 	}
 
+	// No Organization Id
+	if scalewayConfig.OrganizationID == nil {
```
Comment (cbruno10) [inline scaleway/service.go:48]: @jplanckeel What Scaleway resources require this organization ID beside users and API keys? Do any of the existing tables require it?

Up until now, we haven't had any tables that required it, so adding it in now as a required config arg would be a breaking change, i.e., anyone who uses the Scaleway plugin today would suddenly be unable to until they added the `organization_id` config arg.

In some plugins, we add optional config args and then note in the appropriate table docs which optional config args are required, e.g., https://hub.steampipe.io/plugins/turbot/hibp#configuration and https://hub.steampipe.io/plugins/turbot/hibp/tables/hibp_paste.

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@jplanckeel Please see review comments, thanks!

**Comment by cbruno10** [comment]
@jplanckeel Sorry for the extremely long response time, I've added some questions around the new organization ID config arg, can you please have a look when you get a chance? Thanks!

**Pull request merged**